NuKeeper/Engine/Packages/PackageUpdater.cs

```diff
@@ -74,7 +74,7 @@ public class PackageUpdater : IPackageUpdater
 
             foreach (var updateSet in updates)
             {
-                await _updateRunner.Update(updateSet, sources);
+                await _updateRunner.Update(updateSet, sources, null);
```

Comment:
We need to pass in a `IFolder` and not null so we can get the `paket.dependecies` later on.

Reply:
Or some other way we can get access to the dependencies file of Paket